app/src/main/java/net/skyscanner/backpack/demo/stories/DialogStory.kt

```diff
@@ -0,0 +1,142 @@
+package net.skyscanner.backpack.demo.stories
+
+import android.content.Context
+import android.os.Bundle
+import android.view.View
+import net.skyscanner.backpack.button.BpkButton
+import net.skyscanner.backpack.demo.R
+import net.skyscanner.backpack.dialog.BpkDialog
+
+class DialogStory : Story() {
+
+  override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+    super.onViewCreated(view, savedInstanceState)
+    val dialogType = arguments?.getString(DialogStory.TYPE) ?: savedInstanceState?.getInt(DialogStory.TYPE)
+
+    val btn = view.findViewById<BpkButton>(R.id.open_dialog)
+    btn.setOnClickListener({
```

Comment:
IDE warning, move lambda out of parentheses at multiple instances in this file

Reply:
Is not complaining here. I believe you don't have the ktlint config applied locally if it is complaining. I'll remove in any case but this is not supposed to be a problem.